tests/cpp-tests/Classes/Sprite3DTest/Sprite3DTest.cpp

```diff
@@ -786,6 +786,7 @@ void Sprite3DEffectTest::addNewSpriteWithCoords(Vec2 p)
     material->setTechnique("outline_noneskinned");
     sprite->setMaterial(material);
     sprite->setScale(6.f);
+    Director::getInstance()->getTextureCache()->removeUnusedTextures();
```

Comment:
ok. thanks. what about moving that test (or adding a new one) in Texture2d test file?